typescript/cli/src/commands/deploy.ts

```diff
@@ -130,12 +138,8 @@ const warpCommand: CommandModule = {
   command: 'warp',
   describe: 'Deploy Warp Route contracts',
   builder: (yargs) =>
-    yargs.options({
-      config: {
-        type: 'string',
-        description: 'A path to a JSON or YAML file with a warp config.',
-        default: './configs/warp-tokens.yaml',
-      },
+    yargs.options<WarpCommandOptions>({
+      config: warpConfigCommandOption,
```

Comment:
I'm not totally convinced it's worth moving options to the shared options file if they're only used on one place but ndb. It does look neater this way.